src/TextInput/index.js

```diff
@@ -16,6 +21,7 @@ type Props = {|
   +inlineLabel?: boolean,
   +label: string,
```

Comment:
Also I think label should accept React.Node, since we most likely will be passing it our Translation component

Reply:
Hmm, I can change it, but for now we don't have `Translation` component for that cases.

Reply:
Yes, the translation component will (probably) never be a part of orbit, but orbit should not be too strictly flow typed, so we should be able to use our translation component